### PR TITLE
#86 replace deprecated OpaqueToken with InjectionToken

### PR DESCRIPTION
**Pull request by ivankolev**
This should take care of the OpaqueToken deprecation.

### CODE REVIEW

**Comment by ivankolev** [comment]
And I see that there's actually a PR containing this fix already: 
https://github.com/Greentube/localize-router/pull/83
Miroslav, please merge and publish.

File: src/localize-router.config.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import { Inject, OpaqueToken, Provider } from '@angular/core';
+import {Inject, InjectionToken, Provider} from '@angular/core';
 
 /**
  * Guard to make sure we have single initialization of forRoot
- * @type {OpaqueToken<LocalizeRouterModule>}
+ * @type {InjectionToken<LocalizeRouterModule>}
  */
-export const LOCALIZE_ROUTER_FORROOT_GUARD = new OpaqueToken('LOCALIZE_ROUTER_FORROOT_GUARD');
+export const LOCALIZE_ROUTER_FORROOT_GUARD = new InjectionToken('LOCALIZE_ROUTER_FORROOT_GUARD');
```
Comment (peterbakonyi05) [inline src/localize-router.config.ts]: `InjectionToken` is generic, so consider specifying the type: `InjectionToken<LocalizeRouterModule>`. Same for the rest of the tokens :)

**Comment by meeroslav** [comment]
I will actually merge this in, as Nathan's has some obsolete functionality so it needs rework.

File: src/localize-router.config.ts (hunk 1 above)
Comment (ivankolev) [reply src/localize-router.config.ts]: You are right! I will have to create a new PR though, that one is already merged

**Pull request merged**